tf/backends/prod.tf

```diff
@@ -0,0 +1,4 @@
+resource_group_name  = "gratibot-azure-data"
+storage_account_name = "gratibotazuredatatfpd"
+container_name       = "tfstate"
+key                  = "terraform.tfstate"
```

Comment:
Should this be `prod.terraform.tfstate`

Reply:
These are referencing resources in different subscription levels(`var.acr_subscription_id`)

Reply:
These are referencing repos in different subscription levels(_controlled by `var.acr_subscription_id`_)